tests/unit/conftest.py

```diff
@@ -34,8 +35,10 @@ def pyramid_config(pyramid_settings):
 
 @pytest.fixture
 def pyramid_request(
+    db_session,
     pyramid_config,  # pylint:disable=unused-argument
 ):
     pyramid_request = testing.DummyRequest()
     apply_request_extensions(pyramid_request)
+    pyramid_request.db = db_session
```

Comment:
Again I did this manually but it's the same as what the cookiecutter's `conftest.py` template does